src/api/clearlyDefined.js

```diff
@@ -47,8 +48,42 @@ export function harvest(token, spec) {
   return post(url(HARVEST), token, spec)
 }
 
-export function getCuration(token, entity) {
-  return get(url(`${CURATIONS}/${entity.toPath()}`), token)
+export function getCuration(token, entity, params = {}) {
+  const { expandedPrs, pendingPrs } = params
+  return get(
+    url(`${CURATIONS}/${entity.toPath()}`, {
+      expanded: expandedPrs ? 'prs' : null,
+      state: pendingPrs ? 'pending' : null
+    }),
+    token
+  )
+}
+
+/**
+ * List all of the curations (if any) using the given coordinates as a pattern to match
```

Comment:
I feel like this code is duplicated from some other PR or something. The use of boolean params should be discouraged. Use an array of enum values in an `expand` arg or some such. here and above. More extensible and semantically valuable.

Reply:
Part of this PR is used in #320 , due to the refactor of the FullDetailsView components.
I suggest to consider that one, and than update this one when the other one is merged, so we wouldn't have any duplicated implementations.